src/popup/pages/SignTransaction.vue

```diff
@@ -34,11 +37,30 @@ export default defineComponent({
       const { nodeNetworkId, getAeSdk } = useAeSdk();
       const { openDefaultModal } = useModals();
       const { setLoaderVisible } = useUi();
+      const { getLastActiveProtocolAccount } = useAccounts();
+
+      const aeSdk = await getAeSdk();
+      const activeAccountAddress = getLastActiveProtocolAccount(
+        PROTOCOLS.aeternity,
+      )?.address as Encoded.AccountAddress;
+
+      async function replaceCallerTx(tx: Encoded.Transaction) {
+        const unpackedTx = unpackTx(tx) as ReturnType<typeof unpackTx>
+            & { callerId: Encoded.AccountAddress };
```

Comment:
Wrong indentation.

Reply:
eslint didn't seem to complain about it, what would be the correct way?

Reply:
2 spaces.
And yes, eslint is not always complaining about stuff :)

Reply:
done